data_steward/analytics/cdr_ops/raw_rdr_export_qc.py

```diff
@@ -955,5 +955,57 @@
             wear_codes=WEAR_SURVEY_CODES)
 execute(client, query)
 
+# # Check consent_validation for expected number of consent status
+#
+# The 'consent_validation' table is renamed from 'consent' in the rdr import script. This table is used to suppress data in `remove_ehr_data_without_consent.py`. 
+#
+# Each participant should have only one consent status. If this check fails, verify the
+# results and the impact of multiple status records then communicate this finding to the rdr team. Priority is dependant upon the data.<br>
+#
+# The existance of multiple status that are **duplicates** would most likely not affect participant data. <br>
+# The existance of multiple status that are **not duplicates** might affect participant data. See `remove_ehr_data_without_consent.py`
+
+# +
+# Count of participants with multiple validation status' for their ehr consent records.
+query = JINJA_ENV.from_string("""
+
+WITH cte AS (
+SELECT 
+"Have multiple status. These are not duplicates" as issue,
+COUNT(*) AS n
+FROM (SELECT DISTINCT * FROM `{{project_id}}.{{new_rdr}}.consent` )
```

Comment:
Couple changes:
1. consent_validation will be the table name
2. Distinct * will include timestamp as well, and I think there is merit in separating them here (maybe another check?)
```suggestion
FROM (SELECT DISTINCT * EXCEPT (consent_for_electronic_health_records) FROM `{{project_id}}.{{new_rdr}}.consent_validation` )
```

Reply:
I agree, thanks for thinking of that!